src/main/java/delete_event_use_case/DeleteEventPresenter.java

```diff
@@ -0,0 +1,14 @@
+package delete_event_use_case;
+
+/**
+ * @author : Name
+ * @created: November 19th 2022
+ * @since : 2022-11-21, Mon.
+ **/
+public class DeleteEventPresenter implements DeleteEventOutputBoundary {
+    @Override
+    public DeleteEventOutputData prepareSuccessView(DeleteEventOutputData response) {
+        //TODO: View Model Update?
+        return response;
+    }
```

Comment:
You'll need a fail view in case that something goes wrong..... especially while we don't have a complete project